.github/workflows/fabric_acctest.yml

```
@@ -204,24 +206,70 @@ jobs:
           name: pfcr_test_logs
           path: pfcr_test_output.json
 
-      - name: Sweeper PFCR
-        if: ${{ always() }}
-        env:
-          EQUINIX_API_CLIENTID: ${{ secrets.EQUINIX_API_CLIENTID_PFCR }}
-          EQUINIX_API_CLIENTSECRET: ${{ secrets.EQUINIX_API_CLIENTSECRET_PFCR }}
-          METAL_AUTH_TOKEN: ${{ secrets.METAL_AUTH_TOKEN }}
-          SWEEP: "all" #Flag required to define the regions that the sweeper is to be ran in
-          SWEEP_ALLOW_FAILURES: "true" #Enable to allow Sweeper Tests to continue after failures
+  test-STS-creds:
```

The test sweeper step was accidentally deleted when the `test-STS-creds` job was introduced and it will need to be restored.

```suggestion
      - name: Sweeper PFCR
        if: ${{ always() }}
        env:
          EQUINIX_API_CLIENTID: ${{ secrets.EQUINIX_API_CLIENTID_PFCR }}
          EQUINIX_API_CLIENTSECRET: ${{ secrets.EQUINIX_API_CLIENTSECRET_PFCR }}
          METAL_AUTH_TOKEN: ${{ secrets.METAL_AUTH_TOKEN }}
          SWEEP: "all" #Flag required to define the regions that the sweeper is to be ran in
          SWEEP_ALLOW_FAILURES: "true" #Enable to allow Sweeper Tests to continue after failures

  test-STS-creds:
```